MapleServer2/Database/Classes/DatabaseItem.cs

```diff
@@ -218,6 +220,7 @@ private Item ReadItem(dynamic data)
             PairedCharacterId = data.paired_character_id,
             PairedCharacterName = data.paired_character_name,
             PetSkinBadgeId = data.pet_skin_badge_id,
+            RecommendJobs = ItemMetadataStorage.GetRecommendJobs(data.id),
```

Comment:
Isn't this already happening in `Item.SetMetadataValues()`?

Reply:
Nope, `DataseItem` uses the empty constructer for `Item`